branding/css/susemanager-theme.less

```diff
@@ -1314,3 +1314,13 @@ select.small-select {
   align-items: center;
   display: flex;
 }
+
+.fc button.fc-button, .fc button.fc-button-primary {
+  .btn;
+  .btn-default;
+}
+
+.fc-daygrid-day-frame:hover {
+  cursor: pointer;
+  background-color: fadeout(@gray, 70%);
+}
```

Comment:
Please add a comment about the scope of these: these are HACKS to hijack the plugin style to behave like our instead, not classes to reuse.